src/configuration.ts

```diff
@@ -65,4 +69,22 @@ module Lint.Configuration {
             }
         }
     }
+
+    function getDefaultConfiguration(): any {
+        return {
```

Comment:
let's just make this a `const`, right after `CONFIG_FILENAME` on L22. call it `DEFAULT_CONFIG`